src/org/zaproxy/zap/extension/callback/ui/CallbackTable.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2017 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.zaproxy.zap.extension.callback.ui;
+
+import org.jdesktop.swingx.renderer.DefaultTableRenderer;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.view.ZapTable;
+import org.zaproxy.zap.view.renderer.DateFormatStringValue;
+
+import javax.swing.ListSelectionModel;
+import javax.swing.SortOrder;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+import javax.swing.table.TableModel;
+import java.util.Date;
+
+public class CallbackTable extends ZapTable {
```

Comment:
Per previous comment about the `HistoryReferece` this could be changed to a `HistoryReferencesTable` (the table model would have to be changed too).